### DISCUSSION TITLE
Customize markdown code block backgroundcolor and foregroundcolor without lanuage indicator

### DISCUSSION THREAD

**Discussion by jonahfang** [rose-pine/neovim#202] [Q&A]
### Is your feature request related to a problem? Please describe.

No

### Describe the solution you'd like

![image](https://github.com/rose-pine/neovim/assets/16301509/6a606cab-f548-4e06-94f3-2ccda6b58f8b)

I want to change the backgroundcolor and foregroundcolor, how to do it in setup configuration?

### Describe alternatives you've considered

No

**Comment by jonahfang** [comment]
I use rose-pine with neovim treesitter.

**Comment by mvllow** [comment]
The latest updates to treesitter add several new markup groups, try:

```lua
highlight_groups = {
  ["@markup.raw.block.markdown"] = { fg = "...", bg = "..." },
  ["@markup.raw.delimiter.markdown"] = { fg = "...", bg = "..." },
}
```

To find other groups, `:Inspect` will give you the names under your cursor.
